docs/GDI-SOP_github-introduction-for-maintainers.md

```diff
@@ -0,0 +1,193 @@
+
+Generally speaking, if everything works as expected, you will not need to dive much into this repository's workflows. Instead, they will simply aid you with visually appealing icons at Pull Requests:
+- ✔️ - Implies that the **workflow(s) finished correctly** and all checks were passed. 
+- ❌ - Implies that **something went wrong**. This requires further investigation (see example below).
+
+Let's go over a simple example of how a workflow aided in a [recent Pull Request](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pull/38). In this PR, ``M-casado`` created a PR against the ``dev`` branch, which triggered some of the workflows:
+- Compare SOP index table ([``compare_index.yml``](../.github/workflows/compare_index.yml)). It has a ❌, letting us know that something went wrong.
+- Linter of SOPs ([``lint_sops.yml``](../.github/workflows/lint_sops.yml)). It has a ✔️, so it checked the SOPs in this branch and lets us know that they seem to be well formatted.
+
+Notice at the bottom of the image below that, once the owner of the PR noticed the error through this workflow, it was amended and a checkmark ✔️ was obtained.
+![Detecting something went wrong](images/GDI-SOP_github-introduction-for-maintainers_workflow.png)
+
+To see further we can click on the **Details** next to the workflow that failed (see image below). You can inspect for yourself the same page [**here**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/actions/runs/11477389761/job/31939356879). 
+
+In the details of the workflow, you can see that something went wrong at the ``Run SOP index comparison`` step. If we take a closer look, we can see that there is some code executed at this step, and the exit code was ``1`` (i.e., there was an error). Instead of having to dig up exactly why ourselves, the message (in JSON format) that the script prints (see highlighted green below) gives us all the needed clues:
+- There were some differences between the existing [SOP index table](../sops/README.md) and the one that the script created (representing the latest version).
+- The difference was identified for ``GDI-SOP0003`` and specifically at the ``Last modified`` column of the table, where the values were not matching.
+
+![Details of a failed workflow](images/GDI-SOP_github-introduction-for-maintainers_workflow-errors.png)
+
+To fix it was simple, ``M-casado`` updated the index table in the [**next commit**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pull/38/commits/e92709896a20f4ab1cf58d18472e87594cdcdc97), and pushed the changes back to the remote branch (the one with changes), which triggered again the workflow and this time both returned with no errors (✔️).
```

Comment:
Looking at this commit ... it helped me a lot to understand the problem and the solution!